genon/preprocessor/facade/legacy/BOK_첨부용.py

```
@@ -0,0 +1,1866 @@
+                        kwargs.get("whisper_timestamp_granularities", "word")
+                    ),
+                },
+                chunk_sec=int(kwargs.get("whisper_chunk_sec", 29)),
+                chunk_overlap_ms=int(kwargs.get("whisper_chunk_overlap_ms", 300)),
+                tmp_path=tmp_path
+            )
+            vectors = loader.return_vectormeta_format()
+
+            # Remove the temporal chunks
+            try:
+                subprocess.run(['rm', '-r', tmp_path], check=True)
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)